x/intertx/types/v1/init_test.go

```diff
@@ -0,0 +1,8 @@
+package v1
+
+import sdk "github.com/cosmos/cosmos-sdk/types"
+
+func init() {
+	cfg := sdk.GetConfig()
+	cfg.SetBech32PrefixForAccount("regen", "regenpub")
+}
```

Comment:
I think we can remove this file. We have a `config.go` file in the root of `x/intertx` that should apply.

Reply:
Removing this file causing V1 package tests fails with error `expected cosmos got regen` error.